### PR TITLE
Persian translation from Crowdin.

### PR DESCRIPTION
**Pull request by magul**
Made by:
 * Ahmad Karimpour (https://crowdin.com/profile/ahmadina)
 * amir emiram (https://crowdin.com/profile/amiremiram1)
 * Amir Thn (https://crowdin.com/profile/amir75thn)
 * amirfatemi (https://crowdin.com/profile/amirfatemi)
 * Anna Warzecha (vi) (https://crowdin.com/profile/aniav)
 * bahar rahgozar (https://crowdin.com/profile/baharahgozar)
 * Behrouz Pooladrag (https://crowdin.com/profile/iflashlord)
 * Farzan (https://crowdin.com/profile/pharzan)
 * goldjango (https://crowdin.com/profile/goldjango)
 * Hossein Yaghoobi (https://crowdin.com/profile/hosseinyaghoobi1)
 * Maysam Rasheidi (https://crowdin.com/profile/gerdakan.sa)
 * Milad Yarmohammadi (https://crowdin.com/profile/milad.sphinx)
 * Mohamad Torkashvand (https://crowdin.com/profile/torkashvand)
 * Mohammad Dashtizadeh (https://crowdin.com/profile/iMohammad)
 * Mohsen00khani (https://crowdin.com/profile/mohsen00khani)
 * mojtaba sln (https://crowdin.com/profile/mojtaba17sln)
 * Negar Mohamadhasan (https://crowdin.com/profile/nemote95)
 * Nima Karimi (https://crowdin.com/profile/nim4.karimi)
 * olasitarska (https://crowdin.com/profile/olasitarska)
 * Omid (https://crowdin.com/profile/ommmmid)
 * rahimz (https://crowdin.com/profile/rahimz)
 * ramyar khalili (https://crowdin.com/profile/ramyarkhalili)
 * remiba_ (https://crowdin.com/profile/remiba_)
 * Saeid B (https://crowdin.com/profile/saeid6933)
 * Sarvenaz Choobdar (https://crowdin.com/profile/schoobdar)
 * Soroosh Sarabadani (https://crowdin.com/profile/psycho-ir)
 * Z4HRA (https://crowdin.com/profile/Z4HRA)

### CODE REVIEW

**Comment by magul** [comment]
There's still at least one issue to be resolved (as it causes build to fail).

**Comment by magul** [comment]
Current build available here: https://magul.gitbooks.io/django-girls-tutorial/content/v/crowdin-translation-fa/

**Comment by Rahimz** [comment]
I am working on it.

File: fa/GLOSSARY.md [hunk 1]
```diff
@@ -0,0 +1,3 @@
+# ویرایشگر کد
+
+ویرایشگر کد یک برنامه کاربردی است که به شما امکان می دهد کد خود را ذخیره کنید تا بتوانید بعدا به آن بازگردید. شما می توانید در [بخش ویرایشگر کد](./code_editor/README.md) یاد بگیرید که چطور یکی از آن‌ها را دانلود کنید
```
Comment (Rahimz) [inline fa/GLOSSARY.md]:
```suggestion
ویرایشگر کد یک برنامه کاربردی است که به شما امکان می‌دهد کد خود را ذخیره کنید تا بتوانید بعداً به آن بازگردید. شما می‌توانید در [بخش ویرایشگر کد](./code_editor/README.md) یاد بگیرید که چطور یکی از آن‌ها را دانلود کنید
```

Comment (magul) [reply fa/GLOSSARY.md]: @Rahimz that should be applied in Crowdin and then I will update pull request (as we want to manage all translations there eventually).

**Comment by Rahimz** [comment]
I have reviewed the Persian translation completely.

**Comment by magul** [comment]
@Rahimz I've just pushed newest translation from Crowdin. Can you check if that's fine and approve that pull request to be merged?

**Comment by Rahimz** [comment]
I review the Persian translation another time and some typos fixed.

**Comment by Rahimz** [comment]
There are two problems first in the "installation file", the section of "Chromebook setup (if you're using one)"; after click on it, the text does not have any format.
snap_01

**Comment by magul** [comment]
@Rahimz newest version available here: https://magul.gitbooks.io/django-girls-tutorial/content/v/crowdin-translation-fa/

I will check what cause these isssues and would try to provide some solution here.

**Pull request merged**